docs/application/web/guides/multimedia/media-controller.md

```diff
@@ -65,6 +65,11 @@ The main features of the Media Controller API include:
 
   You can [set the media controller server features](#media-controller-server-features) using the server interfaces.
 
+- Sending custom events
+
+  You can [use the server to send events](#send_custom_events) with the `sendEvent()` method.
+
+  To [receive and handle incoming events](#receive_custom_events) in the client, use the `setCustomEventsListener()` method.
```

Comment:
Please make this small change:

To [receive and handle the incoming events](#receive_custom_events) on the client-side, use the `setCustomEventsListener()` method.

Reply:
sure